libs/github.ts

```diff
@@ -9,7 +9,7 @@ export function restGithub<T = any>(path: string, preview = 'hellcat') {
     authorization: `token ${pickGithubToken()}`,
     accept: `application/vnd.github.${preview}-preview+json`
   }
-  const prefixUrl = 'https://api.github.com/'
+  const prefixUrl = process.env.GITHUB_API || "https://api.github.com/"
```

Comment:
Prefer single quotes

```suggestion
  const prefixUrl = process.env.GITHUB_API || 'https://api.github.com/'
```